pkgs/applications/version-management/gitea/default.nix

```diff
@@ -14,12 +14,12 @@
 
 buildGoPackage rec {
   pname = "gitea";
-  version = "1.17.2";
+  version = "1.17.3";
 
   # not fetching directly from the git repo, because that lacks several vendor files for the web UI
   src = fetchurl {
     url = "https://github.com/go-gitea/gitea/releases/download/v${version}/gitea-src-${version}.tar.gz";
```

Comment:
Can you update this to use dl.gitea.io? The pattern is: `https://dl.gitea.io/gitea/${version}/gitea-src-${version}.tar.gz` so that when the move away from github happens, then everything will remain stable.

Reply:
```suggestion
    url = "https://dl.gitea.io/gitea/${version}/gitea-src-${version}.tar.gz";
```

Reply:
Done